docs/15_EgenskaperInformasjonsmodell.adoc

```diff
@@ -0,0 +1,314 @@
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
+
+
+=====  Informasjonsmodell: erstatter
+
+[cols="30s,70d"]
+|===
+|URI|dct:replaces
+|Range|modelldcatno:InformationModel
+|Beskrivelse|Referanse til eldre utgått modell denne modellen er ment å erstatte.
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
+
+
+=====  Informasjonsmodell: er erstattet av
+
+[cols="30s,70d"]
+|===
+|URI|dct:isReplacedBy
+|Range|modelldcatno:InformationModel
+|Beskrivelse|Referanse til oppdatert og nyere modell som erstatter modellen.
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
+
+===== Informasjonsmodell: gyldighetsperiode
+
+[cols="30s,70d"]
+|===
+|URI|dct:temporal
+|Range|dct:PeriodeOfTime
+|Beskrivelse|Modellens gyldighetsintervall.
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
+
+===== Informasjonsmodell: har del
+
+[cols="30s,70d"]
+|===
+|URI|dct:isPartOf
+|Range|modelldcatno:InformationModel
+|Beskrivelse|Referanse til en annen modell som er en del av denne modellen.
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
+
+
+===== Informasjonsmodell: hjemmeside
+
+[cols="30s,70d"]
+|===
+|URI|foaf:homepage
+|Range|foaf:Document
+|Beskrivelse|Brukes til å referere til hjemmesiden til tjenesten.
+|Multiplisitet|0..1
+|Status|Valgfri
+|===
+
+
+===== Informasjonsmodell: modellstatus
+
+[cols="30s,70d"]
+|===
+|URI|adms:status
+|Range|skos:Concept
+|Beskrivelse|Modellens modenhet. Må ha en av verdiene Completed, Deprecated, Under Development, Withdrawn.
+|Multiplisitet|0..1
+|Status|Valgfri
+|===
+
+===== Informasjonsmodell: produsent
+
+[cols="30s,70d"]
+|===
+|URI|dct:creator
+|Range|foaf:Agent
+|Beskrivelse|Referanse til enhet som er produsent av modellen.
+|Multiplisitet|0..1
+|Status|Valgfri
+|===
+
+
+===== Informasjonsmodell: språk
+
+[cols="30s,70d"]
+|===
+|URI|dct:language
+|Range| dct:LinguisticSystem
+|Beskrivelse|Referanse til språket som er brukt i modellen.
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
+
+===== Informasjonsmodell: type
+
+[cols="30s,70d"]
+|===
+|URI|dct:type
+|Range|skos:Concept
+|Beskrivelse|Referanse til typedefinisjoner som kategoriserer modellen og abstraksjonsnivået.
+|Multiplisitet|0..1
+|Status|Valgfri
+|===
+
+===== Informasjonsmodell: utgivelsesdato
+
+[cols="30s,70d"]
+|===
+|URI|dct:issued
+|Range|rdfs:Literal typed as xsd:dateTime
+|Beskrivelse|Dato for den formelle utgivelsen av modellen.
+|Multiplisitet|0..1
+|Status|Valgfri
+|===
+
+
+===== Informasjonsmodell: versjon
+
+[cols="30s,70d"]
+|===
+|URI|owl:versionInfo
+|Range|rdfs:Literal
+|Beskrivelse|Et versjonsnummer eller annen versjonsbetegnelse for modellen.
+|Multiplisitet|0..1
+|Status|Valgfri
+|===
+
+
+===== Informasjonsmodell: versjonsnote
+
+[cols="30s,70d"]
+|===
+|URI|adms:versionNotes
+|Range|rdfs:Literal
+|Beskrivelse|Egenskap som beskriver forskjellene mellom denne og en tidligere versjon av modellen. Kan gjentas for parallelle språkversjoner av versjonsnotater.
+|Multiplisitet                                                  	|0..n
+|Status|Valgfri
+|===
```

Comment:
Creator/produsent - mangler?

Reply:
Den har jeg flyttet oppover (for å ha alfabetisk sortering)